R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -16,6 +16,7 @@
 #
 
 library(testthat)
+library(magrittr)
```

Comment:
I think its better to add a test case that doesn't depend on `magrittr` as our Jenkins environment or other developers might not have the package installed.